models/sma_config.go

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Technologies Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package models
+
+import (
+	"encoding/json"
+)
+
+/*
+ * SetConfig is for SMA processing of incoming requests to set (PUT) Config.
+ *
+ *
+ * SetConfig struct
+ */
+type SetConfig struct {
```

Comment:
Please consider defining this as a formal "request" model, placed in `requests/configuration` for example. The type could then be called `SetConfigRequest`, possibly in either `set.go` or `update.go`. Implement the Validator() interface and provide unit tests please. Example https://github.com/edgexfoundry/go-mod-core-contracts/tree/master/requests/states/admin

Reply:
Yep, I'm on it now 👍